ReservacionDeVuelos/src/components/Checkbox.js

```diff
@@ -0,0 +1,68 @@
+import React, {useState} from 'react';
+import {StyleSheet, View, Text} from 'react-native';
+import colors from '../utils/Colors';
+import CheckBox from '@react-native-community/checkbox';
+import contentText from '../utils/Constants';
+
+function CheckBoxComponent() {
+  const [isSelected, setSelection] = useState(false);
+  const [isSelected2, setSelection2] = useState(false);
```

Comment:
these are bad names

Reply:
Try to be more specific on what component/value does that state affects, e.g "inputNameIsSelected, setInputNameSelection"
"inputEmailIsSelected, setInputEmailSelection"